Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
It was never there AFAIK. I think we need to import Foundation regardless of OS here.